.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this, it is from your VSC

or add line(s) in gitignore

for this

Reply:
removed